src/io/benchmark.zig

```diff
@@ -125,7 +125,7 @@ pub fn main() !void {
 
         fn isRunning(self: Context) bool {
             // Make sure that we're connected
-            if (self.rx.socket.fd == -1) 
+            if (self.rx.socket.fd == -1)
```

Comment:
For a branch like this, we can put this single-line statement also on the same line as the condition, without the newline.

Otherwise, if we need to span lines then we would use a block always.